src/lib/traits.ts

```diff
@@ -5,28 +5,7 @@ export type Category = {
 
 export const CATEGORIES = [
   {
-    key: 'nfts.collection',
+    key: 'mints.title',
     label: 'NFTs',
```

Comment:
change label to 'Zora Minters' maybe? to highlight that we're specifically only including those?